astropy/io/ascii/tests/test_arg_write.py

```diff
@@ -0,0 +1,43 @@
+import warnings
```

Comment:
There is no real need to create an entirely new test file for this one small case.  This can be put into `test_write.py`.